t/copy_input_fastq.t

```diff
@@ -0,0 +1,27 @@
+#!/bin/usr/env perl
+use warnings;
+use strict;
+
+use lib '/Warehouse/Users/csieffert/core-phylogenomics/lib';
+use lib '/Warehouse/Users/csieffert/core-phylogenomics/lib/Stage';
+use Test::More;
+use Test::Exception;
+use Stage::CopyInputFastq;
+use JobProperties;
+use Logger;
+
+#tests that CopyInputFastq.pm is correctly verifying that the input fastq and reference fasta have unique basenames.
+
+my $logger = Logger->new('.');
```

Comment:
Since the logger writes output to files, could you maybe pass a temporary directory here instead of the current directory to avoid cluttering it up with files?  You can use [File::Temp](http://perldoc.perl.org/File/Temp.html) to create a temporary directory.